src/App.js

```diff
@@ -1,16 +1,32 @@
 import React from 'react';
 import './App.css';
+import { useState } from 'react';
+import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
 const App = () => {
+  
+  const [messageData,setMessageData] = useState(chatMessages)
+  
+  const likeMessage = (id) => {
+    setMessageData = messageData => messageData.map(message => {
+      if(message.id === id) {
+        return {...message, liked: message.liked === true}
+      } else {
+        return message;
+      };
+    });
+  };
   return (
     <div id="App">
       <header>
         <h1>Application title</h1>
       </header>
       <main>
-        {/* Wave 01: Render one ChatEntry component
-        Wave 02: Render ChatLog component */}
+        <ChatLog 
+        entries={chatMessages} 
```

Comment:
You should be passing `messageData` down to the `<ChatLog />` component. Otherwise updating the state will have no effect on the data you're currently passing (you're just passing the original data, not the data you added to the component state.